### PR TITLE
SPLAT-698: Periodic test for latest release of provider certification tool on latest published version of OpenShift

### PR DESCRIPTION
**Pull request by bostrt**
Prow CI job and steps for https://github.com/redhat-openshift-ecosystem/provider-certification-tool

### CODE REVIEW

**Comment by bostrt** [comment]
/test pj-rehearse

job killed due to build cluster upgrade

**Comment by bostrt** [comment]
/test pj-rehearse

restarting due to other build cluster upgrade interference

**Comment by bostrt** [comment]
/test pj-rehearse

job failed due to some underlying reason in vsphere (a node rebooted due to mco update)

File: ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+periodics:
+- agent: kubernetes
+  cluster: vsphere
+  cron: 0 0 * * *
```
Comment (bostrt) [inline ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml:4]: 0 0 * * 0

File: ci-operator/config/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  upi-installer:
+    requests:
+      cpu: "3"
+      memory: 5Gi
+tests:
+- as: provider-certification-tool-vsphere
+  cron: 0 0 * * *
```
Comment (bostrt) [inline ci-operator/config/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main.yaml:31]: 0 0 * * 0

Comment (mtulio) [inline ci-operator/config/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main.yaml:31]: What about to run once a week at Sundays?
```suggestion
  cron: 0 0 * * 0
```

File: ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml (hunk 1 above)
Comment (mtulio) [inline ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml:4]: What about to run once a week at Sundays?
```suggestion
  cron: 0 0 * * 0
```

**Review by mtulio — COMMENTED** [review]
Overall lgtm. Detail to run once a week at Sundays (suggestion).

**Comment by mtulio** [comment]
//approve

**Pull request merged**